packages/coding-standards/easy-coding-standard.yml

```diff
@@ -18,6 +18,7 @@ services:
 
     # Shopsys Checkers
     Shopsys\CodingStandards\CsFixer\ForbiddenDumpFixer: ~
+    Shopsys\CodingStandards\CsFixer\ForbiddenPrivateVisibilityFixer: ~
```

Comment:
Yep, the last nitpick note - if I understand correctly, the Fixer does nothing because it is not configured, right? If so, I would prefer to remove this line

Reply:
You're right. Removed.